G/goaws/build_tarballs.jl

```diff
@@ -0,0 +1,34 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "goaws"
+version = v"0.4.5"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/Admiral-Piett/goaws.git", "441c9dc79f38f041a398935b8b53549789fa6100")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/goaws/
+go build -o goaws app/cmd/goaws.go
+mv goaws $prefix/goaws
```

Comment:
minio_jll does this, maybe it will help?
```suggestion
mkdir -p ${bindir}
go build -o ${bindir} app/cmd/goaws.go
```